charts/fluentd-1.14.5/values.yaml

```diff
@@ -0,0 +1,37 @@
+name: verrazzano-fluentd
+
+global:
+  imagePullSecrets: []
+
+image:
+  pullPolicy: IfNotPresent
+  terminationGracePeriodSeconds: 60
+
+logging:
+  credentialsSecret: 'my-secret'
+  usernameKey: username
+  passwordKey: password
+  clusterName: local
+  osURL: 'https://my-opensearch:9200'
+  name: fluentd
+  fluentdImage: container-registry.oracle.com/verrazzano/fluentd-kubernetes-daemonset:v1.14.5-20230922100900-8777b84
```

Comment:
should we fix this and put it under image like we do for other charts?  If so it can be a follow up.

Reply:
I rather not touch it.  This is the value that Verrazzano overrides and it gets properly used.  Changing this would also require changes to the helm templates.

Reply:
Does this need to be fully qualified?  With the `containerRegistry` variable set to OCR, the partially qualifed image name should work correctly.

Reply:
I will push that change and test.

Reply:
Fixed